dtfs-central-api/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Node Alpine Docker Image
-FROM node:23.4.0-alpine3.20
+FROM node:23.9.0-alpine3.20
```

Comment:
should we update these so close to launch or leave for next release?

Reply:
We should be okay with minor release.